src/Commands/Composites.php

```diff
@@ -120,6 +129,22 @@ private function map_composites_by_brand_id($id, callable $callable)
         }
     }
 
+    private function map_composites_by_brand_id_and_source($id, $source, callable $callable)
+    {
+        $compositeQuery = CompositeRepository::find_by_brand_id_and_source($id, $source);
+
+        while ($compositeQuery) {
+            $categories = collect($compositeQuery->edges);
+            $categories->pluck('node')->each(function ($compositeInfo) use ($callable) {
+                $callable(CompositeRepository::find_by_id($compositeInfo->id));
+            });
+            if (isset($compositeQuery->pageInfo->hasNextPage) && $compositeQuery->pageInfo->hasNextPage)
+                $compositeQuery = CompositeRepository::map_composites_by_brand_id_and_source($id, $categories->last()->cursor);
```

Comment:
De her one-liners skriger altså bare på der skal ske fejl, specielt når det er sådan et "stress, hurtigt" script. Men altså, fint nok for nu, det er bare fjollet fremadrettet.